src/typescript/frontend/next.config.mjs

```diff
@@ -1,11 +1,6 @@
 // cspell:word dexscreener
 
 // @ts-check
-import analyzer from "@next/bundle-analyzer";
```

Comment:
Are we sure we want to do this?

Like really really sure?

I mean

I guess it's not a big deal if we do